app/models/manageiq/providers/ansible_tower/inventory/targets/automation_manager.rb

```diff
@@ -0,0 +1,33 @@
+class ManageIQ::Providers::AnsibleTower::Inventory::Targets::AutomationManager < ManagerRefresh::Inventory::Target
```

Comment:
yeah, I changed it to that, since you wanted singular names in my PR

so both dir and class are named Target, Parser, Collector